stackslib/src/net/p2p.rs

```diff
@@ -4218,14 +4220,25 @@ impl PeerNetwork {
 
         let parent_tenure_start_header = NakamotoChainState::get_tenure_start_block_header(&mut chainstate.index_conn(), stacks_tip_block_id, &parent_header.consensus_hash)?
             .ok_or_else(|| {
-                debug!("{:?}: get_parent_stacks_tip: No tenure-start block for parent tenure {} off of child {} (parnet {})", self.get_local_peer(), &parent_header.consensus_hash, stacks_tip_block_id, &parent_block_id);
+                debug!("{:?}: get_parent_stacks_tip: No tenure-start block for parent tenure {} off of child {} (parent {})", self.get_local_peer(), &parent_header.consensus_hash, stacks_tip_block_id, &parent_block_id);
                 net_error::DBError(db_error::NotFoundError)
             })?;
 
+        // TODO: Test this!
```

Comment:
Remove the TODO and add the test, please.  Or, turn this back into a draft.

Reply:
I do believe this is already tested via check_block_heights which would force a refresh of the burnchain view and therefore hit these code paths... I will see if I can more explicitly test it

Reply:
Having a hard time getting a chainstate unit test to work for me. :( Would the integration test be enough?

Reply:
Yes, the integration test would be fine.